WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -201,9 +201,12 @@ private async Task StepConnectionConfirmationPhaseAsync(CancellationToken cancel
 				else if (round.ConnectionConfirmationTimeFrame.HasExpired)
 				{
 					var alicesDidNotConfirm = round.Alices.Where(x => !x.ConfirmedConnection).ToArray();
-					foreach (var alice in alicesDidNotConfirm)
+					if (alicesDidNotConfirm.Length < round.Alices.Count * 0.7)
```

Comment:
I would extract `0.7` to a `const` and write a descriptive comment why we need this.

Reply:
I now use the `MinInputCount`